src/main/java/com/github/netelli/JDBCStuding.java

```diff
@@ -1,45 +1,28 @@
 package com.github.netelli;
 
-import java.sql.*;
+import java.sql.ResultSet;
 
 /**
  * Created by user on 05.12.2016.
  */
 public class JDBCStuding {
     public static void main(String[] args) throws Exception {
-        DBConnection dbConnection = new DBConnection();
-        dbConnection.init();
-        try (Connection connection = dbConnection.getConnection();
-             Statement statement = connection.createStatement()) {
-            createTables(statement);
-            insertData(statement);
+        ProductsDAO productsDAO = new ProductsDAO("jdbc:h2:mem:test");
 
-            ResultSet rs = statement.executeQuery("select * from products");
-            while (rs.next()) {
-                System.out.println(rs.getInt("id") + " : " + rs.getString("title") + " : " + rs.getInt("categoryId"));
-            }
-        }
-    }
+        productsDAO.getConnection();
```

Comment:
better rename this method to init, or set up connection, there is no need to return this connection from class and break encapsulation.